### DISCUSSION TITLE
GitHub student Developer Pack application not working

### DISCUSSION THREAD

**Discussion by adma77ya** [community/community#129399] [GitHub Education]
### Hi everyone,

When applying for github edu program for the first time I hadnt completed my Profile and billing address info.So it got rejected.It said after updating my profile I could log out and then log in and then I could apply for the Program.But now whenever I am uploading a picture and pressing the "Process Application" button it refreshes the page and does not submit the application.I tried this for about 7 times.Its not working.I needed this urgently.
And also theres no upload option, should we always take a pic.

**Comment by akibhaider** [comment]
As I have faced similar issues with my github student pack here are some advice:
1. Try to take more clearer and high resolution photo of your institutonal Identity proof. *That institute must be enlisted to github*
2. Cross check all the information that you have sharede in your billing address and profile. (Something important like correct spelling of your name in the ID(s) or if there's no surname add first name in both of the name field)
3.  Make sure to enable 2FA authentication with at least 2 methods from your profile.
-> Generally, it takes a bit of time to resend another apllication. So, if all these do not help then try re-applying again after (maybe) half an hours interval. Hope it helps!

**Reply by adma77ya** [reply]
I did all this...and tried after a day also...but its not working, whenever I upload a picture and press 'prices application' button it just refreshes the page without submiting my application

**Comment by davevad93** [comment]
Hi @RPMTRaX ,

Take a look at this discussion #126276 , this user had the same issue and solved it by choosing the webcam option instead of uploading the picture.

**Reply by adma77ya** [reply]
But the even after taking a pic using webcam and pressing 'process application ' button it just refreshes the page...really really frustrated with the site man

**Reply by davevad93** [reply]
Hey @RPMTRaX , yeah the application page lately is very buggy, I've seen a lot of similar issues reported here in the discussions. I know that GitHub Education support no longer does personal reviews so try contacting the "default" support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Comment by BartiSzabolcs** [comment]
I have the same issue, the page refreshes itself, even if I use my phone.

**Reply by adma77ya** [reply]
And GitHub doesnt even allow us to propwrly contact help...It has closed personal reviews or some shit...man this felt so frustrating I almost broke and why is it so broken man damn...

**Reply by BartiSzabolcs** [reply]
I have found a solution, I have installed a brand new browser (brave) and used that to submit. It worked. So this is a cache issue perhaps.

**Reply by adma77ya** [reply]
okay will try

**Reply by adma77ya** [reply]
Doesnt work with brave too

**Comment by Valeriucoding** [comment]
I encounter the same issue. I tried desktop, phone and different browsers.

**Reply by adma77ya** [reply]
Man this is frustrating af...GITHUB SUPPORT WHERE ARE YOU

**Reply by Valeriucoding** [reply]
It worked for me an hour ago on chrome. Maybe you can give it another try.

**Reply by adma77ya** [reply]
Not for me😭

**Reply by adma77ya** [reply]
What did u do it on

**Reply by Valeriucoding** [reply]
On desktop, using chrome. But the issue for me is that my webcam doesn't have a good resolution and the document's are not detected well.

**Reply by adma77ya** [reply]
Hmm...thanks man

**Reply by jiesou** [reply]
I tried Chrome, Brave Browser, Firefox, Iceraven on Android and none of them worked

**Reply by Valeriucoding** [reply]
Ur welcome @RPMTRaX. Github support seems to be inexistent.

**Reply by adma77ya** [reply]
GitHub staff replied on one of my posts and told that I could continue normally from now onwards but The things is...its still not working...Is it because the camera is not of good quality or something?

**Reply by Valeriucoding** [reply]
What errors are you getting.

**Reply by adma77ya** [reply]
No errors every time I do it...Its just refreshes...The first time I did it I hadnt completed billing info and profile so it got rejected...the thing is, my id card has my photo and id number on one side and the pass-out year(which is required in the photo) on the other side...so as i can only 1 picture for the id card, this is the problem...So whoch side should i take a pic of...it would help if there was a upload a pic button or something, but we are only allowed to take one single pic

**Reply by Valeriucoding** [reply]
There are some explanations for your case in the pinned posts.
